app/api/private-cloud/products/[licencePlate]/history/route.ts

```diff
@@ -25,8 +25,8 @@ export const GET = apiHandler(async ({ pathParams, session }) => {
     session: session as never,
   });
 
-  if (!requests) {
-    return NotFoundResponse('No requests found for this licence plate.');
+  if (requests.length < 1) {
```

Comment:
I don't think an empty array is `no content`. an empty array is also valid result so that we can return it as it is.
also, the route should be `app/api/private-cloud/products/[licencePlate]/requests/route.ts`